br/pkg/lightning/restore/check_info_test.go

```diff
@@ -39,17 +40,16 @@ import (
 	tmock "github.com/pingcap/tidb/util/mock"
 )
 
-var _ = Suite(&checkInfoSuite{})
-
-type checkInfoSuite struct{}
-
 const passed CheckType = "pass"
 
-func (s *checkInfoSuite) TestCheckCSVHeader(c *C) {
-	dir := c.MkDir()
+func TestCheckCSVHeader(t *testing.T) {
+	dir, err := os.MkdirTemp("", "")
+	require.NoError(t, err)
+	defer os.RemoveAll(dir)
```

Comment:
```suggestion
		dir := t.TempDir()
```

`testing` provides a temp util for testing.

ditto others.